client/me/concierge/shared/closure-notice.js

```diff
@@ -16,43 +17,86 @@ const ClosureNotice = ( { closesAt, displayAt, reopensAt } ) => {
-			}
-		);
-	}
-	return <Card>{ message }</Card>;
+	/** @type {Record<string, { before: string, during: string, reason?: string }>} */
+	const MESSAGES = {
+		default: {
+			before: translate(
+				'{{strong}}Notice:{{/strong}} Quick Start sessions will be closed from %(closesAt)s until %(reopensAt)s. ' +
+					'If you need to get in touch with us, you’ll be able to {{link}}submit a support request{{/link}} ' +
+					'and we’ll get to it as fast as we can. Thank you!',
+				{
+					args: {
+						closesAt: moment.tz( closesAt, guessedTimezone ).format( DATE_FORMAT ),
+						reopensAt: moment.tz( reopensAt, guessedTimezone ).format( DATE_FORMAT ),
+					},
+					components: {
+						link: <a href="/help/contact" />,
+						strong: <strong />,
+					},
+				}
+			),
+			during: translate(
+				'{{strong}}Quick Start Sessions will be closed from %(closesAt)s – %(reopensAt)s for the New Year’s holiday.{{/strong}}{{br/}}' +
+					'If you need to get in touch with us, please submit a {{link}}support request from this page{{/link}} and we will get to it as fast as we can. ' +
+					'Quick Start Sessions will re-open at %(reopensAt)s. Thank you for your understanding!',
+				{
+					args: {
+						closesAt: moment.tz( closesAt, guessedTimezone ).format( DATE_FORMAT ),
+						reopensAt: moment.tz( reopensAt, guessedTimezone ).format( DATE_FORMAT ),
+					},
+					components: {
+						link: <a href="/help/contact" />,
+						strong: <strong />,
+						br: <br />,
+					},
+				}
+			),
+		},
+		gm: {
+			before: translate(
+				'{{strong}}Note:{{/strong}} Support sessions will not be available between %(closesAt)s and %(reopensAt)s.',
+				{
+					args: {
+						closesAt: moment.tz( closesAt, guessedTimezone ).format( DATE_FORMAT ),
+						reopensAt: moment.tz( reopensAt, guessedTimezone ).format( DATE_FORMAT ),
+					},
+					components: {
+						strong: <strong />,
+					},
+				}
+			),
+			during: translate(
+				'{{strong}}Note:{{/strong}} Support sessions are not available before %(reopensAt)s.',
+				{
+					args: {
+						reopensAt: moment.tz( reopensAt, guessedTimezone ).format( DATE_FORMAT ),
+					},
+					components: {
+						strong: <strong />,
+					},
+				}
+			),
+			reason: translate(
+				'Why? Once a year, the WordPress.com Happiness Engineers and the rest of the WordPress.com family get together to work on improving our services, building new features, and learning how to better serve our customers like you. But never fear! If you need help in the meantime, you can submit an email ticket through the contact form: {{contactLink}}https://wordpress.com/help/contact{{/contactLink}}',
+				{
+					components: {
+						contactLink: <a href="/help/contact" />,
+					},
+				}
+			),
+		},
+	};
+
+	const variant = MESSAGES[ isGM ? 'gm' : 'default' ];
+
+	return (
+		<Card>
+			<VStack>
+				<div>{ currentDate.isBefore( closesAt ) ? variant.before : variant.during }</div>
+				{ variant.reason && <div>{ variant.reason }</div> }
+			</VStack>
```

Comment:
I would've just used two `<p>`s, but there is a global margin-bottom on paragraph tags that makes it look bad 😭 Going with this here for simplicity.